ref/Arithmetic_Operators.md

```diff
@@ -300,14 +300,20 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and 1: `random()`
 
     > random()
-    < 0.4680764123124367
-    ~ 0\.\d{4,}
+    < 0.468
+    ~ 0\.\d+
+    > 1000*random()
+    < 278.4083
+    ~ \d+\.\d+
 
 #### (0,1)-normally distributed random number: `randomnormal()`
 
     > randomnormal()
-    < 0.1325114717517828
-    ~ -?\d+\.\d{4,}
+    < 0.1325
+    ~ -?\d+\.\d+
```

Comment:
No. Although the standard normal distribution is *concentrated* around zero, there is in fact no bound on its values, large magnitudes are just pretty unlikely. Unless I made a mistake in [my computation](http://www.wolframalpha.com/input/?i=erfc%281%2Fsqrt%282%29%29%2F2), you have about 16% chance for a value less than -1 and again 16% for greater 1.